cvat-ui/src/components/annotation-page/top-bar/annotation-menu.tsx

```diff
@@ -188,6 +191,18 @@ function AnnotationMenuComponent(props: Props & RouteComponentProps): JSX.Elemen
             <Menu.Item key={Actions.LOAD_JOB_ANNO}>Upload annotations</Menu.Item>
             <Menu.Item key={Actions.EXPORT_JOB_DATASET}>Export job dataset</Menu.Item>
             <Menu.Item key={Actions.REMOVE_ANNOTATIONS}>Remove annotations</Menu.Item>
+            <Menu.Item
+                key={Actions.RUN_ACTIONS}
+                onClick={() => {
+                    const div = window.document.createElement('div');
+                    window.document.body.append(div);
```

Comment:
Why do we need empty div?

Reply:
Because ReactDOM.render replaces all the content of element where we embed something. 
If you look at modals in .antd, they are implemented the same way (in a wrapper)